drivers/osdp/Kconfig.pd

```diff
@@ -0,0 +1,217 @@
+	  setting causes the PD to energize the Output during the inactive state and
+	  de-energize the Output during the active state. This feature allows the
+	  support of "fail-safe/fail-secure" operating modes.
+	  - 03: Like 01, plus: The PD is able to accept timed commands to the
+	  Output. A timed command specifies the state of the Output for the
+	  specified duration.
+	  - 04: Like 02 and 03 - normal/inverted drive and timed operation.
+
+config OSDP_PD_CAP_OUTPUT_CONTROL_NUM_ITEMS
+	int "Number of items"
+	default 0
+	help
+	  The number of Outputs.
+
+endmenu # "Output Control"
+
+menu "LED Control"
+
+config OSDP_PD_CAP_READER_LED_CONTROL_COMP_LEVEL
+	int "Compliance Level"
+	default 0
+	help
+	  Possible values:
+	  - 01: the PD support on/off control only
+	  - 02: the PD supports timed commands
+	  - 03: like 02, plus bi-color LEDs
+	  - 04: like 02, plus tri-color LEDs
+
+config OSDP_PD_CAP_READER_LED_CONTROL_NUM_ITEMS
+	int "Number of items"
+	default 0
+	help
+	  The number of LEDs per reader.
+
+endmenu # "LED Control"
+
+menu "Audible Output"
+
+config OSDP_PD_CAP_READER_AUDIBLE_OUTPUT_COMP_LEVEL
+	int "Compliance Level"
+	default 0
+	help
+	  Possible values:
+	  - 01: the PD support on/off control only
+	  - 02: the PD supports timed commands
+
+config OSDP_PD_CAP_READER_AUDIBLE_OUTPUT_NUM_ITEMS
+	int "Number of items"
+	default 0
+	help
+	  The number of audible annunciators per reader
+
+endmenu # "Audible Output"
+
+menu "Text Output"
+
+config OSDP_PD_CAP_READER_TEXT_OUTPUT_COMP_LEVEL
+	int "Compliance Level"
+	default 0
+	help
+	  Possible values:
+	  - 00: The PD has no text display support
+	  - 01: The PD supports 1 row of 16 characters
+	  - 02: the PD supports 2 rows of 16 characters
+	  - 03: the PD supports 4 rows of 16 characters
+	  - 04: TBD.
+
+config OSDP_PD_CAP_READER_TEXT_OUTPUT_NUM_ITEMS
+	int "Number of items"
+	default 0
+	help
+	  Number of textual displays per reader
+
+endmenu # "Text Output"
+
+menu "Card Data Format"
+
+config OSDP_PD_CAP_CARD_DATA_FORMAT_COMP_LEVEL
+	int "Compliance Level"
+	default 0
+	help
+	  Possible values:
+	  - 01: the PD sends card data to the CP as array of bits, not exceeding
+	  1024 bits.
+	  - 02: the PD sends card data to the CP as array of BCD characters, not
+	  exceeding 256 characters.
+	  - 03: the PD can send card data to the CP as array of bits, or as an
+	  array of BCD characters.
+
+config OSDP_PD_CAP_CARD_DATA_FORMAT_NUM_ITEMS
+	int
+	default 0
+	help
+	  N/A, set to 0.
+
+endmenu # "Card Data Format"
+
+menu "Time Keepinig"
+
+config OSDP_PD_CAP_TIME_KEEPING_COMP_LEVEL
+	int "Compliance Level"
+	default 0
+	help
+	  Possible values:
+	  - 00: The PD does not support time/date functionality
+	  - 01: The PD understands time/date settings per Command osdp_TDSET
+	  - 02: The PD is able to locally update the time and date
+
+config OSDP_PD_CAP_TIME_KEEPING_NUM_ITEMS
+	int
+	default 0
+	help
+	  N/A, set to 0.
```

Comment:
This looks like unnecessary option if the value is always 0. Perhaps do a plain define instead?

Reply:
Done.